src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calc($cell-size * 7 + $gap * 6);
+  gap: $gap;
```

Comment:
The variable `$gap` is used here. Ensure it is defined in your SCSS files to avoid compilation errors.